providers/fips/fipsprov.c

```diff
@@ -22,10 +22,25 @@
 #include "internal/evp_int.h"
 
 /* Functions provided by the core */
-static OSSL_core_get_param_types_fn *c_get_param_types = NULL;
-static OSSL_core_get_params_fn *c_get_params = NULL;
-static OSSL_core_put_error_fn *c_put_error = NULL;
-static OSSL_core_add_error_vdata_fn *c_add_error_vdata = NULL;
+static OSSL_core_get_param_types_fn *c_get_param_types;
+static OSSL_core_get_params_fn *c_get_params;
+static OSSL_core_put_error_fn *c_put_error;
+static OSSL_core_add_error_vdata_fn *c_add_error_vdata;
```

Comment:
Why did you remove the initialization?

Reply:
That was a seperate PR .

Reply:
Because of the C guarantee to initialize to NULL.